.gitlab/functional_test/system_probe.yml

```diff
@@ -11,17 +11,36 @@
     - .kitchen_datadog_agent_flavor
   rules:
     !reference [.on_system_probe_changes_or_manual]
+  needs:
+    job: get_file_diff
+    artifacts: true
   stage: functional_test
   variables:
     AGENT_MAJOR_VERSION: 7
     DD_PIPELINE_ID: $CI_PIPELINE_ID-fnct
     CHEF_VERSION: 14.15.6
   script:
+    - set +e
+    - export SYSTEM_PROBE_PATHS=$(echo "$SYSTEM_PROBE_SRC_PATH")
```

Comment:
Can we put this under `.on_system_probe_changes_or_manual` and reference it here and other places using [!reference](https://docs.gitlab.com/ee/ci/yaml/yaml_optimization.html#reference-tags) tags?

Reply:
Also was `$SYSTEM_PROBE_SRC_PATH` not available as is here?

Reply:
I had some issues before with multiline gitlab variable to shell for loop, but I'll try again to be sure. And yea using a reference is good idea doing that o7

Reply:
Done 👍